src/groups/mqb/mqbblp/mqbblp_queueengineutil.cpp

```diff
@@ -397,6 +404,14 @@ void QueueEngineUtil::logRejectMessage(
         << "deleted." << MWCTSK_ALARMLOG_END;
 }
 
+bsls::Types::Int64 QueueEngineUtil::getMessageQueueTime(
```

Comment:
In my previous PR, I turned this `getMessageQueueTime` to a local function, because it's  not used outside the `QueueEngineUtil` cpp file. If it's still not used outside, let's keep it hidden from `QueueEngineUtil` interface.